perun-core/src/test/java/cz/metacentrum/perun/core/impl/DatabaseManagerImplIntegrationTest.java

```diff
@@ -0,0 +1,67 @@
+package cz.metacentrum.perun.core.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import cz.metacentrum.perun.core.AbstractPerunIntegrationTest;
+import cz.metacentrum.perun.core.api.DBVersion;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.bl.DatabaseManagerBl;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+/**
+ * @author Simona Kruppova 410315
+ * @author Oliver Mrazik 410035
+ */
+public class DatabaseManagerImplIntegrationTest extends AbstractPerunIntegrationTest{
```

Comment:
These tests would probably cause problems on real DB. From the SQL I'm not sure, that they correctly cleanup when they finish in error/ok or before they start. Also SQL seems to be valid for postgres/hsqldb only (e.g. oracle doesn't know column type text). I'll test it locally. Quick solution is to run these tests only in a "default" profile using class/test annotation:

```java
@ActiveProfiles("default")
public class DatabaseManagerImplIntegrationTest extends AbstractPerunIntegrationTest {
```